include/parcels.h

```diff
@@ -142,3 +143,20 @@ static inline int parcels_randint(int low, int high)
 {
   return (rand() % (high-low)) + low;
 }
+
+static inline float parcels_normalvariate(float loc, float scale)
```

Comment:
Some type of docstring would be nice here, since the details of this are not immediately obvious.